torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -320,7 +320,7 @@ Tensor VariableType::detach(const Tensor & self) const {
   return std::move(result);
 }
 
-Tensor & VariableType::detach_(Tensor & self) const {
+Tensor & VariableType::detach_(Tensor & (*_op)(Tensor &), Tensor & self) {
```

Comment:
This is pretty weird, especially since you never actually call `_op` here. Are you sure this is the pattern you want for "Variable-only" operations?

Reply:
I guess this is fine.